site/src/examples/tag/TagInCard.tsx

```diff
@@ -0,0 +1,35 @@
+import { ReactElement } from "react";
+import { Card, FlexLayout, H3, StackLayout, Tag, Text } from "@salt-ds/core";
+import styles from "./index.module.css";
+
+export const TagInCard = (): ReactElement => (
+  <Card
+    className={styles.category3}
+    style={{
+      width: 315,
+    }}
+    accent="top"
+  >
+    {/* in your css:
```

Comment:
May need whitespace better here?

Not sure whether we should actively promote override component tokens

Reply:
I'll remove the comment since it is to do with card and not with tag, i think if we were to add category to card then we could use category here instead of the styling, but given it is already on the site, i would be inclined to keep it as it is (css hidden) and refactor it once we remove/update the card one